lib/galaxy/tool_util/verify/asserts/archive.py

```diff
@@ -49,11 +50,19 @@ def _list_from_zip(bytes, path):
     return sorted(lst)
 
 
-def assert_has_archive_member(output_bytes, path, verify_assertions_function, children, all="false", n: int = None, delta: int = 0, min: int = None, max: int = None, negate: bool = False):
+def assert_has_archive_member(output_bytes: bytes,
+                              path: str,
+                              verify_assertions_function: Callable,
+                              children: List[etree.Element],
```

Comment:
You sure that's the only type that can be passed here ? I think this should also work with yaml test definitions

Reply:
Good point. Seems that the parser makes it a List[Dict] anyway:

https://github.com/galaxyproject/galaxy/blob/32a41700f3026aa24b151a3fd7e7205ee3fd5fee/lib/galaxy/tool_util/parser/xml.py#L741
https://github.com/galaxyproject/galaxy/blob/32a41700f3026aa24b151a3fd7e7205ee3fd5fee/lib/galaxy/tool_util/parser/yaml.py#L282

Will change it.